### DISCUSSION TITLE
Cannot scroll `less` by mouse wheels

### DISCUSSION THREAD

**Discussion by QuarticCat** [microsoft/terminal#14076] [Q&A] [answered]
My WT version is 1.14.2281.0

`less` runs on a remote Linux machine connected by SSH and its version is 608.

I found #9165, where @DHowett said it's a duplicate of #381, which is fixed by #12561, which has been released in `Windows Terminal Preview v1.14.143`. I think my local WT version includes this fix but mouse scrolling still doesn't work for `less`.

Mouse scrolling can work if I add a `--mouse` flag but correspondingly I cannot select text using mouse then.

**Comment by j4james** [comment, marked as answer]
For scrolling to work in `less` you also need to have the _alternate scroll_ mode enabled, which `less` doesn't do automatically. But you can do that yourself with something like `printf "\e[?1007h"` from a bash shell.

We should probably have that mode enabled by default though (that's tracked in issue #13187).
